### PR TITLE
New article: Fastify and Preact for quick web app prototyping

### PR DESCRIPTION
**Pull request by lmammino**
See article preview [here](https://github.com/lmammino/loige.co/blob/new-fastify-article/content/posts/2019-02-16_fastify-and-preact-for-quick-web-app-prototyping/index.md).

Reviews and early comments, most welcome (pleaseee)!

😋

### CODE REVIEW

File: content/posts/2019-02-16_fastify-and-preact-for-quick-web-app-prototyping/index.md [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+
+The [official Fastify documentation](https://www.fastify.io/docs/) is a great place to start from, if you want to find out more about this wonderful framework.
+
+_Full disclosure_: I am a member of the core team and I mostly contribute with building and maintaining the Fastify website and its documentation.
+
+## Preact
+
+If you never heard about **Preact**, it is a UI library for the web, initially created as a lighter and faster drop-in alternative to React by [Jason Miller](https://github.com/developit).
+
+The project has been quite successful and it's now co-maintained by [Marvin Hagemeister](https://github.com/marvinhagemeister) with more than 20.000 GitHub stars and 160 contributors.
```
Comment (marvinhagemeister) [inline content/posts/2019-02-16_fastify-and-preact-for-quick-web-app-prototyping/index.md]: Thanks for the shoutout! Should note that both @andrewiggins and @ForsakenHarmony put an equal amount of work into Preact. I'm just more visible as I post more frequently on twitter 👍

Comment (developit) [reply content/posts/2019-02-16_fastify-and-preact-for-quick-web-app-prototyping/index.md]: We should set up a proper page listing contributors.
<details>
<summary>partial list</summary>

</details>

Comment (lmammino) [reply content/posts/2019-02-16_fastify-and-preact-for-quick-web-app-prototyping/index.md]: I'll update the article and include them as well

Comment (marvinhagemeister) [reply content/posts/2019-02-16_fastify-and-preact-for-quick-web-app-prototyping/index.md]: Adding them all and listing each contributor is probably too much for an article like this. Maybe something along the lines of `...and is now maintained by a team of contributors` or something like that would read more fluently?

Comment (lmammino) [reply content/posts/2019-02-16_fastify-and-preact-for-quick-web-app-prototyping/index.md]: Good point, thanks :)

**Pull request merged**